docs(cli): Add missing/remove unsupported commands from `README` (#7544)
keptn/keptn: cli/README.md

where **command**, **entity**, **name**, and **flags** are:

- **command**: Specifies the operation that you want to perform, for example, install, create, send.

- **entity**: Specifies the entity type. For example, the following commands run a create, get, and update operation on the project, service, and domain entity:

    ```console
    keptn create project 

>     keptn get service

Reviewer: Can you please change it to `keptn create service`?

Author: I've changed it here: 93acf2c0be3d412c6224482a53681c45ba5bb0e3!

>     keptn configure domain

Reviewer: Can you please remove this command?

Author: Removed here: 93acf2c0be3d412c6224482a53681c45ba5bb0e3!

where **command**, **entity**, **name**, and **flags** are:

> - **command**: Specifies the operation that you want to perform, for example, install, create, send.

Reviewer 2: ```suggestion
- **command**: Specifies the operation that you want to perform, such as install, create, or send.

```

Author: Thanks! I've made the adjustment here: 47bac16388b6e5aaaf846b536eb47cddc7a090f2

| Command  | Description  |
|:---:|---|

> | `abort`  | Aborts the execution of a sequence |

Reviewer 2: I might favor using the imperative for the verb in the description, so this would be "Abort the execution of a sequence".  It makes the prose a little tighter and saves a space, always good in a table.  The current form is not wrong, of course -- this is just my preference.

Author: Hi @StackScribe, I agree with using the imperative for the descriptions. Perhaps this is a change that can be made in another issue along with an adjustment to the `keptn help` command, as I've matched the descriptions in the README the command's output. Specifically, they match the following:

https://user-images.githubusercontent.com/42628024/164546056-6e626cb3-e0a1-4a7c-a4ce-8fd19b1051aa.png

Reviewer 2: Sounds good!

| `add-resource`  | Adds a local resource to a service within your project in the specified stage |
| `auth`  | Authenticates the Keptn CLI against a Keptn installation  |
| `completion`  | Generate completion script  |
| `configure`  | Configures one of the specified parts of Keptn  |
| `create`  | Creates a new project, service or secret |
| `delete`  | Deletes a project |
| `generate`  | Generates the markdown CLI documentation or a support archive |
| `get`  | Displays an event or Keptn entities such as project, stage, or service |
| `help`  | Help about any command |
| `install`  | Installs Keptn on a Kubernetes cluster |
| `pause`  | Pauses the execution of a sequence |
| `resume`  | Resumes the execution of a sequence |
| `send`  | Sends an event to Keptn |
| `set`  | Sets flags of the CLI configuration |
| `status`  | Checks the status of the CLI |
| `trigger`  | Triggers the execution of an action in keptn |
| `uninstall`  | Uninstalls Keptn from a Kubernetes cluster |
| `update`  | Updates an existing Keptn project |
| `upgrade`  | Upgrades Keptn on a Kubernetes cluster |
| `version`  | Shows the version of Keptn and Keptn CLI |

## Examples: Common operations

> Use the following set of examples to help you familiarize yourself with running the commonly used `keptn` operations:

Reviewer 2: ```suggestion
Use the following set of examples to familiarize yourself with running the commonly used `keptn` operations:

```

Author: Thanks, I've made the adjustment here: 47bac16388b6e5aaaf846b536eb47cddc7a090f2
